DashKit.xcodeproj/project.pbxproj

```diff
@@ -988,6 +988,11 @@
 					"$(PROJECT_DIR)/Carthage/Build/iOS",
 					"$(PROJECT_DIR)/dash-sdk-ios/frameworks",
 				);
+				LOCALIZED_STRING_MACRO_NAMES = (
+					NSLocalizedString,
+					CFCopyLocalizedString,
+					LocalizedString,
+				);
```

Comment:
[nit] Should just the project be updated and other targets use the project build settings?